src/Module_Nodes/parse_node_definition.cpp

```diff
@@ -431,6 +433,99 @@ static auto find_outputs(std::string const& text, NodeDefinition_Data& res)
     );
 }
 
+template<typename T>
+void modify_description(const std::string& name_property, Cool::InputDefinition<T>& input, const std::string& param_description)
+{
+    if (input.name == name_property)
+    {
+        input.description = param_description;
+    }
+}
+
+std::string get_rid_of_triple_slashs(const std::string& text, const size_t& last)
```

Comment:
"slashes" instead of "slashs"

Reply:
I just deleted the function wich is is no longer usefull, we do it directly